src/main/java/ai/pluggy/client/response/Investment.java

```diff
@@ -7,34 +7,38 @@
 public class Investment {
 
   String id;
+  String itemId;
+  String code;
+  String isin;
   String number;
   String name;
-  Double balance;
-  String currencyCode;
+  String owner;
   InvestmentType type;
   InvestmentSubtype subtype;
+  String currencyCode;
+  Double rate;
+  String rateType;
+  Double fixedAnnualRate;
   Double annualRate;
   Double lastTwelveMonthsRate;
-  String itemId;
-  String code;
+  Double lastMonthRate;
```

Comment:
Double type in java has known issues (https://stackoverflow.com/questions/179427/how-to-resolve-a-java-rounding-double-issue), I think that is better to use BigDecimal instead